### PR TITLE
Added saved location

### PR DESCRIPTION
**Pull request by yuliya-polyuganych**

### CODE REVIEW

File: Car.Backend/Car.Data/Constants/Constants.cs [hunk 1]
```diff
@@ -28,5 +28,7 @@ public static class Constants
         public const double MaxLatitude = 90;
         public const double MinLongitude = -180;
         public const double MaxLongitude = 180;
+
```
Comment (OleksandrAntoniuk) [inline Car.Backend/Car.Data/Constants/Constants.cs]: please remove an extra line

File: Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Car.Data.Constants;
+using Car.Domain.FluentValidation;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
```
Comment (OleksandrAntoniuk) [inline Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs:4]: please remove nunit dependency

File: Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        [Fact]
+        public void LocationName_IsNotValid_GeneratesValidationError()
+        {
+            var longName = new string('*', Constants.LocationNameMaxLength + 1);
+            validator.ShouldHaveValidationErrorFor(locationModel => locationModel.Name, longName);
+        }
+
+        [Xunit.Theory]
```
Comment (OleksandrAntoniuk) [inline Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs]: simplify with Theory only

**Review by OleksandrAntoniuk — CHANGES_REQUESTED** [review]

File: Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs (hunk 2 above)
Comment (yuliya-polyuganych) [reply Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs:4]: we use this dependency in all places for TestFixture end other attributes

File: Car.Backend/Car.Data/Constants/Constants.cs (hunk 1 above)
Comment (yuliya-polyuganych) [reply Car.Backend/Car.Data/Constants/Constants.cs]: done

File: Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs (hunk 3 above)
Comment (yuliya-polyuganych) [reply Car.Backend/Car.UnitTests/FluentValidationTests/Location/CreateLocationModelValidatorTest.cs]: done

**Review by YuriiDanyliv — APPROVED** [review]

**Review by vovazavislyak — APPROVED** [review]

**Review by EdgeOfMordor — APPROVED** [review]

**Pull request merged**